Sources/TabBar/View/TabBar.swift

```diff
@@ -96,30 +96,32 @@ public struct TabBar<TabItem: Tabbable, Content: View>: View {
             .frame(maxWidth: .infinity)
         }
     }
+
     
-    public var body: some View {
-        ZStack {
-            self.content
-                .frame(maxWidth: .infinity, maxHeight: .infinity)
-                .environmentObject(self.selectedItem)
-            
-            GeometryReader { geometry in
-                VStack {
-                    Spacer()
-                    
-                    self.tabBarStyle.tabBar(with: geometry) {
-                        .init(self.tabItems)
-                    }
-                }
-                .edgesIgnoringSafeArea(.bottom)
-                .visibility(self.visibility)
-            }
-        }
-        .onPreferenceChange(TabBarPreferenceKey.self) { value in
-            self.items = value
-        }
-    }
-    
+	public var body: some View {
+		GeometryReader { geometry in
+			ZStack {
+				self.content
+					.environmentObject(self.selectedItem)
+					.padding(.bottom, 50)
+					.frame(maxWidth: .infinity, maxHeight: .infinity)
+				
+				VStack {
+					Spacer()
+					
+					self.tabBarStyle.tabBar(with: geometry) {
+						.init(self.tabItems)
+					}
+				}
+				.edgesIgnoringSafeArea(.bottom)
+				.visibility(self.visibility)
+			}
+			.onPreferenceChange(TabBarPreferenceKey.self) { value in
+				self.items = value
+			}
+		}
+		.ignoresSafeArea(.keyboard)
```

Comment:
Could you explain why we need this here?

Reply:
If you use a keyboard inside a child view of tabbar. The tabbar moves up with keyboard without this modifier. This is the reason I increased iOS version from 13 to 14.